src/main/java/calculator/domain/CalculateNumber.java

```diff
@@ -0,0 +1,31 @@
+package calculator.domain;
+
+public class CalculateNumber {
+    private static final String VALID_EMPTY_NUMBER_MESSAGE = "숫자 값이 없습니다.";
+    private static final String VALID_NO_NUMBER_MESSAGE = "숫자가 아닙니다.";
+
+    private int number;
```

Comment:
이 값은 변하지 않을 값이니 final로 지정하면 어떨까요?

```suggestion
    private final int number;
```